### DISCUSSION TITLE
leaderboard gold/platinum division border > 4000 threshold

### DISCUSSION THREAD

**Discussion by bcambl** [BattlesnakeOfficial/feedback#203] [Bug Reports]
There is a minor cosmetic issue with the summer league 2022 leader board where the boarder that separates the platinum and gold division is appearing _above_ 4000 instead of 3999.
![Screenshot from 2022-06-25 23-31-33](https://user-images.githubusercontent.com/5565939/175820235-000c2d67-7fbd-4a2e-b1cd-024d5f3dd504.png)

**Comment by chris-bsnake** [comment]
Thanks for letting us know @bcambl, I'll throw a bug in our tracking system

**Comment by ambadoom** [comment]
I don't think this is a bug although it is a bit confusing. The border represents who qualifies for the platinum tournament and that is limited to 16(I think) snakes. Any more snakes that are >= 4000 points are considered platinum in that they have the badge but they don't make it into the tournament so they are below the line

**Comment by chris-bsnake** [comment]
While that is true, the size limit doesn't really come into play until we have over 32 snakes with 4000+ points.  I did look at the code and there is some logic in the display that is doing a `>` instead of '>=' when checking the points, so a bug does exist and i'll be putting in a fix today
